D:SWARM-Use-Case-at-SLUB-Dresden.md

```diff
@@ -0,0 +1,59 @@
+At [SLUB Dresden](http://www.slub-dresden.de), the *Streaming version* of d:swarm is used for normalising data retrieved from various sources before feeding it to a [Solr](http://lucene.apache.org/solr/) server. Clients such as the new [SLUB Katalog (beta)](http://katalogbeta.slub-dresden.de/) can then request data from Solr and present it to library users.
+
+[[img/use-case-slub.png]]
+
+## Data Sources and Formats
+
+The following data sources are currently processed with d:swarm at SLUB Dresden:
+
+* [deutsche FOTOTHEK](http://www.deutschefotothek.de)
+  * OAI-PMH-XML (containing elements from the [DC-Elements](http://purl.org/dc/elements/1.1/) and [Europeana](http://europeana.eu/terms) namespace)
```

Comment:
"containing elements" -> "containing records that make use of terms from the DC-Elements and Europeana vocabularies" (?)